components/availability-select/timeslot-rnd.tsx

```diff
@@ -12,6 +12,8 @@ import { ResizeDirection } from 're-resizable';
 import dynamic from 'next/dynamic';
 import useTranslation from 'next-translate/useTranslation';
 
+import CloseIcon from 'components/icons/close-icon';
```

Comment:
This file name should be `components/icons/close.tsx` (not `close-icon.tsx`) for naming consistency. Also, using `icon` in the filename is redundant because it is already located in the `icons` directory.